python-package/xgboost/core.py

```diff
@@ -1066,8 +1067,9 @@ def update(self, dtrain, iteration, fobj=None):
             self.boost(dtrain, grad, hess)
 
     def boost(self, dtrain, grad, hess):
-        """
-        Boost the booster for one iteration, with customized gradient statistics.
+        """Boost the booster for one iteration, with customized gradient
+        statistics.  Like update, This function should not be called
```

Comment:
Small typo
```suggestion
        statistics.  Like update, this function should not be called
```

Also we might want to add a [cross-ref](https://stackoverflow.com/a/21297186/209882) here to clearly show which update we're talking about.